test/unit/auth/credential.spec.ts

```diff
@@ -395,6 +408,24 @@ describe('Credential', () => {
         privateKey: mockCertificateObject.private_key,
       });
     });
+
+    it('should parse valid token if app def creds point to default refresh token loc', () => {
```

Comment:
Lets spell out words in full: `should create a RefreshTokenCredential if application default credentials variable point to gcloud credentials path`.